prime-router/settings/organizations-local.yml

```diff
@@ -331,6 +331,14 @@
         topic: covid-19
         jurisdictionalFilter:
           - matches(ordering_facility_state, NM)
+        fileName:
+          lowerCase: true
+          elements:
+            - cdcprime_
+            - receivingOrganization()
+            - _test_
+            - createdDate()
+            - .hl7
```

Comment:
I'm concerned about filename uniqueness.   It looks like the default createdDate is really a timestamp, which should be good enough, but what if someone set up the same fileNameTemplate but with createdDate(yyyyMMdd) - this would be bad.    Given our future expected volumes, we should assume many files each day to each state.

Should we enforce having enough fields to guarantee uniqueness?  Like, if you use createdDate(yyyyMMdd), then you also have to include uuid(), or something like that.

Reply:
This is already going to be a problem for Santa Clara and California as they have the same file name standard, and there's nothing that distinguishes which goes with which. We may need to end up adding folders into our blob storage, one for each receiver to stave this off a little longer.